### ISSUE TITLE
Update example 13, aria-haspopup is no longer global

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/aria#1183]
Under "Presentational Roles Conflict Resolution", it says:

> For example, aria-haspopup is a global attribute and would always be applied; aria-level is not a global attribute and would therefore only apply if the element was not in a presentational state.

Since aria-haspopup is no longer a global attribute, this examples should be updated.

**Comment by JAWS-test** [comment]
For this issue there is already a PR: https://github.com/w3c/aria/pull/852

**Comment by carmacleod** [comment]
The ARIA Working Group just discussed `Update example 13, aria-haspopup is no longer global`.

<details><summary>The full IRC log of that discussion</summary>
&lt;carmacleod> TOPIC: Update example 13, aria-haspopup is no longer global<br>
&lt;carmacleod> github: https://github.com/w3c/aria/issues/1183<br>
&lt;carmacleod> mck: It is not recommended to use aria-label on a heading - so example goes against APG<br>
&lt;carmacleod> mck: change the heading content to match APG, saying something like "when heading content is insufficient..."<br>
&lt;carmacleod> mck: want an example that demonstrates that the semantic is valid, but role presentation is ignored<br>
&lt;carmacleod> carmacleod: ok, I'll fix up the example, will ping james when done<br>
</details>
